doc/sql/integer.rst

```diff
@@ -0,0 +1,30 @@
+Integer Operations
+==================
+
+There are four supported integer datatypes, ``TINYINT``, ``SMALLINT``,
```

Comment:
Can we also state (if possible) how many bits each integer type has?

Reply:
That is in the section on types. But I can repeat that.